core/state_processor.go

```diff
@@ -517,7 +515,6 @@ func InitSignerInTransactions(config *params.ChainConfig, header *types.Header,
 		go func(from int, to int) {
 			for j := from; j < to; j++ {
 				types.CacheSigner(signer, txs[j])
```

Comment:
Keep cache hash for Tx
```suggestion
				types.CacheSigner(signer, txs[j])
				txs[j].CacheHash()
```

Reply:
Re-added the `CacheHash`, PTAL